plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftQueryFailureRecoverySmokeTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.redshift;
+
+import io.trino.operator.RetryPolicy;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static io.trino.execution.FailureInjector.InjectedFailureType.TASK_GET_RESULTS_REQUEST_FAILURE;
+
+public class TestRedshiftQueryFailureRecoverySmokeTest
+        extends BaseRedshiftFailureRecoveryTest
+{
+    public TestRedshiftQueryFailureRecoverySmokeTest()
+    {
+        super(RetryPolicy.QUERY);
+    }
+
+    @Override
+    @DataProvider(name = "parallelTests", parallel = true)
+    public Object[][] parallelTests()
+    {
+        // Intentionally skip the regular FTE tests to execute the smoke test faster
```

Comment:
everything we do intentionally is intentional, and having a code comment is sufficient to document our sanity.
omit the word "Intentionally"